assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -59,9 +54,13 @@ const fetchSaveUserInputSettingsStore = createFetchStore( {
 // Actions
 const SET_USER_INPUT_SETTINGS = 'SET_USER_INPUT_SETTINGS';
 const SET_USER_INPUT_SETTING = 'SET_USER_INPUT_SETTING';
+const START_SAVING_USER_SETTINGS = 'START_SAVING_USER_SETTINGS';
+const FINISH_SAVING_USER_SETTINGS = 'FINISH_SAVING_USER_SETTINGS';
```

Comment:
I don't think we need these and all the new logic around them. The fetch store automatically creates `START_FETCH_SAVE_USER_INPUT_SETTINGS` and `FINISH_FETCH_SAVE_USER_INPUT_SETTINGS` to check whether settings are being saved.

With your original approach where the display of this screen conditionally includes the "Congrats" notification, you probably needed the changes you added here, but since this isn't necessary I don't think we need those separate indicators.

Reply:
Yep, makes sense, removed.